### PR TITLE
Updated Knex version and added migrations directory for managing data…

### PR DESCRIPTION
**Pull request by SpartanDeBoss**
…base schema changes

# Description

Please provide a short summary explaining what this PR is about.

Fixes # (issue)

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: packages/server/migrations/20240321154546_create_team_members_table.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.hasTable('TeamMembers').then((exists) => {
```
Comment (DanJecu) [inline packages/server/migrations/20240321154546_create_team_members_table.js:6]: Do you guys really need this table? Maybe `employees` table could have another column.

**Review by DanJecu — CHANGES_REQUESTED** [review]
It looks good, good job 👏.

I'm leaving some small comments 👀

File: packages/server/migrations/20240321154546_create_team_members_table.js (hunk 1 above)
Comment (nishadipri) [reply packages/server/migrations/20240321154546_create_team_members_table.js:6]: The purpose of this TeamMembers table is to keep records of the team members, as well as to make it easy to record if one person becomes a part of a different team in the future.

Comment (DanJecu) [reply packages/server/migrations/20240321154546_create_team_members_table.js:6]: Ok, I understand now.

Maybe we you guys need to add some foreign keys here as well 🤔

What do you think @nishadipri @SpartanDeBoss

File: packages/server/config/database.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// const mysql = require('mysql');
```
Comment (orhantoy) [inline packages/server/config/database.js:1]: Why is all of this being commented out? Is the file needed or not?

Comment (SpartanDeBoss) [reply packages/server/config/database.js:1]: > Why is all of this being commented out? Is the file needed or not?

I was supposed to delete that, but unfortunately, I forgot it and then pushed it. @orhantoy

**Pull request merged**